tests/PHPStan/Parser/PhpDocParserTest.php

```diff
@@ -1538,15 +1595,16 @@ public function provideMultiLinePhpDocData(): array
 				'/**
 				  *
 				  *
-				  * @param Foo $foo 1st multi world description
+				  * @param Foo $foo 1st multi world description with empty lines
```

Comment:
Appended to help differentiate to catch an in-development bug where it was consuming all empty line breaks between the next `Lexer::TOKEN_IDENTIFIER`

Reply:
the empty lines must be retained (because they are significant when phpdoc contains e.g. markdown)